public/contextMenu.js

```diff
@@ -0,0 +1,258 @@
+/*
```

Comment:
**Comment unrelated to line number**

I know we don't have a linter on this project yet, but I think we should refactor this entire file to be more consistent with the code style from the other files by using 2 spaces indentation

Reply:
Good point! The file was refactored.